File: Clases_Csharp/Mascota.cs
using System.Collections.Generic;

namespace SistemaVeterinaria
{
    public class Mascota
    {
        public string IdMascota { get; private set; }
        public string Nombre { get; private set; }
        public string Especie { get; private set; }
        public string Raza { get; private set; }
        public string Edad { get; private set; }
        public List<Consulta> HistorialClinico { get; private set; }

        // El constructor ahora recibe 'string edad'
        public Mascota(string idMascota, string nombre, string especie, string raza, string edad)
        {
            IdMascota = idMascota;
            Nombre = nombre;
            Especie = especie;
            Raza = raza;
            Edad = edad;
            HistorialClinico = new List<Consulta>();
        }

        public void AgregarConsulta(Consulta consulta)
        {
            HistorialClinico.Add(consulta);
        }
    }
}

File: Clases-Bluej/Mascota.java
import java.util.ArrayList;

public class Mascota {
    private String idMascota;
    private String nombre;
    private String especie;
    private String raza;
    private int edad;
    private ArrayList<Consulta> historialClinico;

    public Mascota(String idMascota, String nombre, String especie, String raza, int edad) {
        this.idMascota = idMascota;
        this.nombre = nombre;
        this.especie = especie;
        this.raza = raza;
        this.edad = edad;
        this.historialClinico = new ArrayList<>();
    }

    public void agregarConsulta(Consulta consulta) {
        this.historialClinico.add(consulta);
    }

    public String getIdMascota() { return idMascota; }
    public String getNombre() { return nombre; }
    public String getEspecie() { return especie; }
    public String getRaza() { return raza; }
    public int getEdad() { return edad; }
    public ArrayList<Consulta> getHistorialClinico() { return historialClinico; }
}